src/main/resources/configuration.properties

```diff
@@ -19,6 +19,7 @@ parentlevel=10
 hunkanalysis=false
 #
 min_nb_hunks=1
+min_nb_commit=1
```

Comment:
I would say it's more `max_nb_commit_to_analyze` and the default should be 100.

WDYT?

Reply:
I meant that only the commits after the min_nb_commit th commit are analyzed by that, and I set the default to analyze all the commits (==1) because I thought maybe we have only one commit. 

should I change it to get the number of commits we want to analyze instead of the number of commits skipped?

Reply:
> should I change it to get the number of commits we want to analyze instead of the number of commits skipped?

yes, it would be better